apps/web-giddh/src/app/shared/header/components/group-update/group-update.component.ts

```diff
@@ -489,15 +492,18 @@ export class GroupUpdateComponent implements OnInit, OnDestroy, AfterViewInit {
     }
 
     public getAccountFromGroup(groupList: IGroupsWithAccounts[], uniqueName: string, result: boolean): boolean {
-        groupList.forEach(el => {
-            if (el && el.accounts) {
-                if (el.uniqueName === uniqueName && (el.category === 'income' || el.category === 'expenses')) {
-                    result = true;
-                    return;
+        groupList.forEach(element => {
+            if (element && element.accounts) {
+                if (element.uniqueName === uniqueName) {
+                    let isDebCred = this.isDebtorCreditorGroup(uniqueName);
+                    if (element.category === 'income' || element.category === 'expenses' || isDebCred) {
+                        result = true;
+                        return;
```

Comment:
Please check this return statement will return `undefined` value from this method to the caller whereas `result` is set to `true` in this `if` condition